credentialz/gnsi-credentialz.yang

```diff
@@ -48,18 +48,56 @@ module gnsi-credentialz {
         description
             "SSH server credentials freshness-related data.";
 
-        leaf active-trust-bundle-version {
+        leaf active-trusted-user-ca-keys-version {
             type version;
             description
-                "The version of the bundle of the Certificate Authority
-                certificates a.k.a. trust bundle.";
+                "The version of the Certificate Authority keys.";
```

Comment:
A grammar question from a not-native speaker: does 'the Certificate Authority keys' cover the case of multiple CAs? Or just multiple keys from one CA?

Reply:
Ha, from a native speaker I'm not even sure. I think either "Certificate Authority keys" or the possessive "Certificate Authority's keys" would be acceptable (but the latter sounds bad to me). "keys" clearly implies multiple are possible, but yes the singular "Certificate Authority" seems like it could be interpreted as being only one - although it sort of doesn't make sense to have multiple keys for one CA as literally the definition of a CA for openssh is just a keypair that signs stuff - much less complicated than x509.

I have no better name I can think of offhand.